experimental/dds/tree2/src/feature-libraries/editable-tree-2/api/object.ts

```diff
@@ -0,0 +1,47 @@
+import { FieldKey } from "../../../core";
+import { AllowedTypes, StructSchema } from "../../typed-schema";
+import { TypedNodeUnion } from "../editableTreeTypes";
+import { getProxyForField } from "./node";
+
+export function createObjectProxy<TTypes extends AllowedTypes>(
+	content: TypedNodeUnion<TTypes>,
+	schema: StructSchema,
+) {
+	// To satisfy 'deepEquals' level scrutiny, the target of the proxy must be an object literal in order
+	// to pass 'Object.getPrototypeOf' comparison.
```

Comment:
I think I understand this comment, but it would be more clear if it mentioned what the alternative is to passing an object literal (the alternative that _doesn't_ pass `Object.getPrototypeOf` comparison).